### PR TITLE
Further information about AR platform support installation and update of packages versions

### PR DESCRIPTION
**Pull request by Kent1LG**
## Overview
The ARFoundation CrossPlatform documentation is a bit confusing about the setting up of the scene. 
I added a side note about the fact that you can manually install the ARFoundation components to take advantage of this framework strengths.

## Changes
- Related to an issue I did #8261

### CODE REVIEW

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 1]
```diff
@@ -10,15 +10,22 @@
...
+    **Unity 2019.4.x**
 
     | **Android** | **iOS** |
     | --- | --- |
-    | AR Foundation  <br/> Version: 2.1.4 |  AR Foundation  <br/> Version: 2.1.4 |
-    | ARCore XR Plugin <br/> Version: 2.1.2 | ARKit XR Plugin <br/> Version: 2.1.2 |
+    | AR Foundation  <br/> Version: 2.1.8 |  AR Foundation  <br/> Version: 2.1.8 |
+    | ARCore XR Plugin <br/> Version: 2.1.11 | ARKit XR Plugin <br/> Version: 2.1.9 |
+
+    **Unity 2020.1.x**
```
Comment (wiwei) [inline Documentation/CrossPlatform/UsingARFoundation.md:23]:
```suggestion
    **Unity 2020.1.x (Not currently formally supported, included for informational purposes only)**
```

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 2]
```diff
@@ -50,6 +57,11 @@ The following steps presume use of the MixedRealityToolkit object. Steps require
 
     For more information about configuring the Unity AR camera settings provider: [Unity AR camera settings provider](../CameraSystem/UnityArCameraSettings.md).
 
+> [!NOTE]
+>This installation check (when the application start) if there is the AR Foundation components on the scene, if not, they are automaticaly add to make it work with ARCore and ARKit.
+>If you need to set a specific behaviour, you should add the components you need by yourself. 
+>For more information about AR Foundation components ands installation, check this [documentation](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@2.2/manual/index.html#samples).
+
```
Comment (wiwei) [inline Documentation/CrossPlatform/UsingARFoundation.md:64]:
```suggestion
> [!NOTE]
> This installation check (when the application start) if there is the AR Foundation components on the scene, if not, they are automaticaly add to make it work with ARCore and ARKit.
> If you need to set a specific behaviour, you should add the components you need by yourself. 
> For more information about AR Foundation components ands installation, check this [documentation](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@2.2/manual/index.html#samples).

```

**Comment by wiwei** [comment]
Thanks @Kent1LG!

Change lgtm, modulo a comment about 2020.1 because we don't formally support that yet (i.e. I think with some work that @keveleigh is doing things will build and stuff)

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 3]
```diff
@@ -10,15 +10,22 @@
 
     | **Android** | **iOS** | Comments |
     | --- | --- | --- |
-    | AR Foundation  <br/> Version: 1.5.0 - preview 6 | AR Foundation  <br/> Version: 1.5.0 - preview 6 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
+    | AR Foundation  <br/> Version: 1.5.0 - preview 7 | AR Foundation  <br/> Version: 1.5.0 - preview 7 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
```
Comment (keveleigh) [inline Documentation/CrossPlatform/UsingARFoundation.md:13]: >Version: 1.5.0 - preview 7

@CDiaz-MS @davidkline-ms I feel like I remember we had been running into issues between preview 6 and preview 7?

Comment (CDiaz-MS) [reply Documentation/CrossPlatform/UsingARFoundation.md:13]: I remember that the preview version did have an impact on iOS testing.  

@Kent1LG  Has the AR Foundation preview 7 version been tested on both Android and iOS?

Comment (Kent1LG) [reply Documentation/CrossPlatform/UsingARFoundation.md:13]: Hi Kurtis and Catherine ! Thank you for this point, I didn't try IOS yet, I'm sorry. This is a matter of time, I'm gonna handle this tomorrow (French timing ahah) as soon as possible.
I keep you all in touch!

Comment (Kent1LG) [reply Documentation/CrossPlatform/UsingARFoundation.md:13]: Hi again, so, I just tried. Unfortunatly ArFoundation can't be use in 1.5 preview 7 versions. I tried preview 6 instead on both and this works fine.

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 4]
```diff
@@ -50,6 +57,11 @@ The following steps presume use of the MixedRealityToolkit object. Steps require
 
     For more information about configuring the Unity AR camera settings provider: [Unity AR camera settings provider](../CameraSystem/UnityArCameraSettings.md).
 
+> [!NOTE]
+> This installation check (when the application start) if there is the AR Foundation components on the scene, if not, they are automaticaly add to make it work with ARCore and ARKit.
```
Comment (keveleigh) [inline Documentation/CrossPlatform/UsingARFoundation.md:61]:
```suggestion
> This installation checks (when the application starts) if the AR Foundation components are in the scene. If not, they are automatically added to make it work with ARCore and ARKit.
```

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 5]
```diff
@@ -50,6 +57,11 @@ The following steps presume use of the MixedRealityToolkit object. Steps require
...
+> If you need to set a specific behaviour, you should add the components you need by yourself. 
+> For more information about AR Foundation components ands installation, check this [documentation](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@2.2/manual/index.html#samples).
```
Comment (keveleigh) [inline Documentation/CrossPlatform/UsingARFoundation.md:63]:
```suggestion
> For more information about AR Foundation components and installation, check this [documentation](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@2.2/manual/index.html#samples).
```

**Comment by wiwei** [comment]
Horray

**Pull request merged**